### DISCUSSION TITLE
Why aren't the headings bolded? (2.30.1 on Debian 12.8)

### DISCUSSION THREAD

**Discussion by EH86055** [fastfetch-cli/fastfetch#1412] [Q&A]
Hi. For some reason, the headings of each category aren't bold in Konsole (first image). They are bold in Kitty (second image). The font I'm using does have a bold version, and other things in Konsole can be bold. How might I fix this? Thanks in advance for your help.

![Screenshot_20241124_213612](https://github.com/user-attachments/assets/61204be2-9eca-47be-b94b-f3eaebd79f9c)
![Screenshot_20241124_213859](https://github.com/user-attachments/assets/449576a7-c201-44e3-aca4-dac0a7676c23)

[config.jsonc](https://github.com/user-attachments/files/17892899/config.jsonc)

**Comment by CarterLi** [comment]
![image](https://github.com/user-attachments/assets/083ce2d9-887f-44e0-b782-eed0372b4fd2)

**Reply by EH86055** [reply]
Hi, thanks for the fast response. Unfortunately Konsole does draw bold font, just not for Fastfetch. The box is already checked.
![Uploading Screenshot_20241125_060411.png…]()

**Reply by CarterLi** [reply]
Try `fastfetch --bright-color`

**Reply by EH86055** [reply]
I forgot to mention that ... it just changes the colour of the text, still not bold. It might be a problem with Konsole and not Fastfetch.
